wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/MessageToolboxView.swift

```diff
@@ -281,9 +278,8 @@ final class MessageToolboxView: UIView {
                 self.statusSeparatorLabel.isHidden = true
                 self.countdownLabel.isHidden = true
             }
-        case .reactions(let reactionsString):
+        case .reactions:
```

Comment:
Perhaps we should remove this as a datasource.

Reply:
I was thinking about the same thing. Let me check if it impacts any other view and I'm planning to remove it. @johnxnguyen.